tests/components/FieldValueDirectAnswer.test.tsx

```diff
@@ -0,0 +1,155 @@
+    render(<FieldValueDirectAnswer result={result}/>);
+    expect(screen.getByRole('link', { name: '[value]' }))
+      .toHaveAttribute('href', 'https://www.instagram.com/[value]');
+  });
+
+  it('contains proper url href for field type "TwitterHandle"', () => {
+    const result = {
+      ...fieldValueDAResult,
+      fieldType: BuiltInFieldType.TwitterHandle
+    } as FieldValueDirectAnswerType;
+    render(<FieldValueDirectAnswer result={result}/>);
+    expect(screen.getByRole('link', { name: '@[value]' }))
+      .toHaveAttribute('href', 'https://twitter.com/[value]');
+  });
+
+  it('contains proper url href for field type "ComplexURL"', () => {
+    const result = {
+      ...fieldValueDAResult,
+      fieldType: BuiltInFieldType.ComplexURL,
+      value: {
+        url: '[url]',
+        displayUrl: '[preferUrl]',
+        preferDisplayUrl: false,
+      }
+    } as FieldValueDirectAnswerType;
+    render(<FieldValueDirectAnswer result={result}/>);
+    expect(screen.getByRole('link', { name: '[url]' }))
+      .toHaveAttribute('href', '[url]');
+  });
+
+  it('displays the preferred displayUrl for field type "ComplexURL"', () => {
+    const result = {
+      ...fieldValueDAResult,
+      fieldType: BuiltInFieldType.ComplexURL,
+      value: {
+        url: '[url]',
+        displayUrl: '[preferUrl]',
+        preferDisplayUrl: true,
+      }
+    } as FieldValueDirectAnswerType;
+    render(<FieldValueDirectAnswer result={result}/>);
+    expect(screen.getByRole('link', { name: '[preferUrl]' }))
+      .toHaveAttribute('href', '[url]');
+  });
+
+  it('contains proper url href for field type "Phone"', () => {
+    const result = {
+      ...fieldValueDAResult,
+      fieldType: BuiltInFieldType.Phone,
+      value: '123-456-7890'
+    } as FieldValueDirectAnswerType;
+    render(<FieldValueDirectAnswer result={result}/>);
+    expect(screen.getByRole('link', { name: '123-456-7890' }))
+      .toHaveAttribute('href', 'tel:123-456-7890');
+  });
+
+  it('contains proper url href for field type "Email"', () => {
+    const result = {
+      ...fieldValueDAResult,
+      fieldType: BuiltInFieldType.Email,
+      value: [
+        'email1@yext.com',
+        'email2@yext.com'
+      ]
+    } as FieldValueDirectAnswerType;
+    render(<FieldValueDirectAnswer result={result}/>);
+    expect(screen.getByRole('link', { name: 'email1@yext.com' }))
+      .toHaveAttribute('href', 'mailto:email1@yext.com');
+    expect(screen.getByRole('link', { name: 'email2@yext.com' }))
+      .toHaveAttribute('href', 'mailto:email2@yext.com');
+  });
+
+  it('displays the preferred address for field type "Address"', () => {
+    const extraDescription = 'This is the preferred address to display';
+    const address: Address = {
+      line1: '61 Ninth Avenue',
+      line2: '14th Floor',
+      city: 'New York',
+      region: 'NY',
+      postalCode: '10011',
+      countryCode: 'US',
+      extraDescription
+    };
+    const result = {
```

Comment:
maybe the result data for the different field types can also be moved to fixtures? I think it could be reused in the stories then?

Reply:
mm most of them are a bit different, maybe example emails and number. I think it's fine to keep them separate?

Reply:
ok, just checking!